awx/ui_next/src/components/CodeMirrorInput/VariablesDetail.jsx

```diff
@@ -111,7 +112,7 @@ function VariablesDetail({ dataCy, helpText, value, label, rows, fullHeight }) {
             css="color: var(--pf-global--danger-color--100);
             font-size: var(--pf-global--FontSize--sm"
           >
-            Error: {error.message}
+            <Trans>Error:</Trans> {error.message}
```

Comment:
Did you confirm this text renders ok? We've had some issues with <Trans> contents not displaying in the past

Reply:
It does, yes